core/src/backend/ui.rs

```diff
@@ -330,11 +394,7 @@ impl Default for NullFileDialogResult {
     }
 }
 
-impl FileDialogResult for NullFileDialogResult {
-    fn is_cancelled(&self) -> bool {
-        true
-    }
-
+impl FileSelection for NullFileDialogResult {
```

Comment:
I think it would be better to rename `NullFileDialogResult` to `NullFileSelection` for consistency.